package.json

```diff
@@ -216,6 +214,7 @@
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.5.5",
     "@lavamoat/allow-scripts": "^1.0.6",
+    "@lavamoat/preinstall-always-fail": "^1.0.0",
```

Comment:
Could this be omitted entirely? Since it's a dependency of `@lavamoat/allow-scripts` already. It still serves the same purpose when it's a transitive dependency.

Reply:
Removed